### PR TITLE
feat(): add caution while using template bound functions

### PR DESCRIPTION
**Pull request by eduardoRoth**
Add a caution message about performance when using template bound functions and the default change strategy

### CODE REVIEW

File: content/components/minimize-logic-in-templates.md [hunk 1]
```diff
@@ -39,3 +39,7 @@ export class SomeComponent {
   }
 }
 ```
+
+### Caution
```
Comment (d3lm) [inline content/components/minimize-logic-in-templates.md:43]:
```suggestion
# Gotchas
```

File: content/components/minimize-logic-in-templates.md [hunk 2]
```diff
@@ -39,3 +39,7 @@ export class SomeComponent {
...
+
+Be careful when doing this while the `ChangeDetectionStrategy` is set as `Default`, as it'd cause the template bound functions to be called each time the `Change Detection Cycle` runs. Optimize this by using the `async` pipe and having `observables` that return the required value (also while using `OnPush` Change Detection Strategy.
```
Comment (d3lm) [inline content/components/minimize-logic-in-templates.md:45]:
```suggestion
Be careful when the `ChangeDetectionStrategy` is set to `Default`, as it'd cause the functions bound in the template to be called each time the `Change Detection Cycle` runs. You can optimize this by turning on the `OnPush` change detection strategy and leverage the `async` pipe in combination with `Observable`s that return the desired value.
```

**Review by d3lm — CHANGES_REQUESTED** [review]
Awesome! Suggested some rewording. Let me know what you think.

File: content/components/minimize-logic-in-templates.md [hunk 3]
```diff
@@ -39,3 +39,7 @@ export class SomeComponent {
   }
 }
 ```
+
+### Gotchas
```
Comment (geromegrignon) [inline content/components/minimize-logic-in-templates.md:43]: I would consider renaming it to 'Best practices' as 'Gotchas' might confuse non-native English speakers not being used to seeing such a term used as a title (Gotchas having different meanings).

Comment (d3lm) [reply content/components/minimize-logic-in-templates.md:43]: `Best Practices` sounds good to me 👍

**Comment by d3lm** [comment]
@eduardoRoth Do you want to update the PR so we can get it merged?

**Comment by eduardoRoth** [comment]
Sure, I'll update it today

**Comment by eduardoRoth** [comment]
@d3lm updated the sub header.

**Review by d3lm — APPROVED** [review]

**Pull request merged**